lib/mongo/session.rb

```diff
@@ -770,7 +779,7 @@ def add_txn_num!(command)
     def add_txn_opts!(command, read)
       command.tap do |c|
         # The read preference should be added for all read operations.
-        if read && txn_read_pref = txn_read_preference
+        if false && read && txn_read_pref = txn_read_preference
```

Comment:
Why is this block being disabled? if we don't want it to run, can we just remove it?